code/index.html

```diff
@@ -2,21 +2,108 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>&#127846 CupcakeMania</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!-- Link for google fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
+    <!-- Header with logo and navbar -->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <a href="#">
+        <img class="menu" src="./Images/menu_icon.svg" alt="menu image">
+      </a>
+      <div class="logo"></div>
```

Comment:
🤔

Reply:
Hi Matilda, the feedback is a little unclear here, but I guess you mean that I should rename my classes?

Reply:
So sorry! I was thinking the image was the cupcake and that this was just an empty `div`. I misunderstood and I should've made a clearer comment than an emoji!